README.md

```diff
@@ -185,6 +195,48 @@ Remark that only the network part should be specified here! When specifying a cl
 
 Based on the idea and examples detailed at <https://linuxmonk.ch/wordpress/index.php/2016/managing-dns-zones-with-ansible/> for the gdnsd package, the zone files are fully idempotent, and thus only get updated if "real" content changes.
 
+### Zone types and Zone type auto-detection
+
+Zone `type` is an optional zone parameter that defines if the zone type should be of `primary`, `secondary` or `forward` type. When `type` parameter is ommited, zone type will be autodetected based on the intersection of host IP addresses and `primaries` record when configuring primary or secondary zone. When `primaries` is not defined and `forwarders` is defined, the zone type will be set to `forward`.
+
+Zone autodetection functionality is especially usefull when deploying multi-site DNS infrastrucutre. It is convinient to have a "shared" `bind_zones` definitions in a single group inventory file for all dns servers ( ex. `group_vars\dns.yml`). Such an approach allows to switch between primary and secondary server(s) roles by updating `primaries` record only and rerunning the playbook.
```

Comment:
Hi Gregory,  I have just returned from a break, to a flurry of emails, and some awesome work by yourself to streamline this role.  So I'd also like to offer thanks in this regard...

I noted a couple of spelling corrections in the README, 
convinient - convenient
omited - omitted

Reply:
Go it. I will make sure to add spell checker plugin to my editor :). I will address all spelling mistakes shortly.